bot.py

```diff
@@ -32,6 +32,7 @@ def __init__(self):
         self.handlers = {}
         self.voice_client = None
         self.play_ctx_queue = queue.Queue()
+        self.block_after = False
```

Comment:
Instead of block_after, how about should_autoplay_next_song (and then flip the truth values everywhere)?